README.md

```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
+<div class="alert alert-warning">
+  This integration is only supported in the US East (us-east-1) region.
+</div>
+{{< /site-region >}}
+
```

Comment:
The us-east-1 region is for customers own amazon account, not our regions. The integration can be used to send data to any of our DC.

Reply:
Additionally, this region restriction is only for the public registry. Anybody can still use the zip packages we provide to privately register the extensions in any region.

Reply:
```suggestion
{{< site-region region="us3,eu,gov" >}}
<div class="alert alert-warning">
  This integration is only supported in the US East (us-east-1) region of the AWS Management Console UI.
</div>
{{< /site-region >}}

```

Reply:
I don't think I would phrase it like this. More like "Datadog Cloudformation resources are only available on the cloudformation public registry in the us-east-1 region. You can still use the provided packages to privately register a resource in any region."
Also show this all the time, not only for `us3,eu,gov` as those are DD regions and it's unrelated to availability.
Does that make sense ?

Reply:
@zippolyte one more time!